src/App.css

```diff
@@ -1,28 +1,32 @@
 .App {
-    text-align: center;
+	text-align: center;
```

Comment:
I generally prefer spaces to tabs, but this is not a big complaint as the codebase is consistent.